error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/StaticImport.java

```diff
@@ -189,7 +190,8 @@ public final class StaticImport extends BugChecker implements MemberSelectTreeMa
           "INSTANCE",
           "newBuilder",
           "of",
-          "valueOf");
+          "valueOf",
+          "values");
```

Comment:
... but `ThirdPartyLibrary.values()`, _would_ be statically imported, which is not something we want.

Reply:
This should also be reverted.